src/utils/http.ts

```diff
@@ -1,28 +1,33 @@
 import axios from "axios";
 
-let baseURL = "";
+// build the axios api base url
+let beApi: any = {};
 
 export function setBaseUrl(url: string) {
-  baseURL = url;
+  beApi = axios.create({
+    baseURL: url,
+    timeout: 15000 // timeout so we're not getting ECONNRESET
```

Comment:
That's pretty long. Is that a reasonable timeout?

Reply:
yeah, its a bit long. we can shorten it. what do you think 5 seconds?